terraform/variables.tf

```diff
@@ -173,3 +179,14 @@ variable "distributed_tff_example_worker_2_address" {
   description = "Address of the second worker of the distributed TensorFlow Federated example. Set this when the worker is outside the coordinator mesh."
   type        = string
 }
+
+variable "cross-device" {
+  description = "Enable cross device infrastructure deployment"
+  type        = bool
+  default     = false
+}
+
+variable "spanner_instance_config" {
+  description = "Configuration of Spanner instance"
```

Comment:
Extend this description because users don't know what this is about.

Reply:
Done